composer.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "spryker-eco/heidelpay",
+  "description": "Heidelpay bundle",
+  "license": "proprietary",
+  "require": {
+    "heidelpay/php-api": "dev-master",
+    "heidelpay/php-customer-messages": "dev-master",
```

Comment:
You probably want to lock this down to actual releases before releasing!

Reply:
@mykyta65536 please use tags instead of dev-master (https://github.com/heidelpay/php-api/releases)